get_ci_vm/README.md

```diff
@@ -1,10 +1,151 @@
+      path set in the `$SRCDIR` env. var. (see above).  It *may* be (highly recommended)
+      mounted with the `O` overlay option to protect local contents.  It's assumed
+      the repository is in the state desired by the user to reproduce on the remote VM.
+      Any commited or uncommited changes will be preserved and transfered as-is.
+   1. The `$HOME/.config/gcloud` directory *must* exist, and be mounted inside
+      the container at `/root/.config/gcloud` (with the `z` option).
+      Repository-specific GCE settings and (sensitive) credentials are stored here.
+      Though they will be maintained using destinct configuration profile names,
+      contents should be protected as if they were passwords.  i.e. Never commit them
+      into any container image or provide them publically.
+   1. The `$HOME/.config/gcloud/ssh` directory *must* exist, and be mounted
+      inside the container at `/root/.ssh` (with the `z` option).  This directory
+      will hold auto-maintained ssh keys for accessing VMs in both GCE and EC2.
+      Removing these files is generally safe, as they will be re-generated on
+      next execution.
+   1. The `$HOME/.aws` directory *must* exist, and be mounted inside the container
+      at `/root/.aws` (with the `z` option).  This directory will contain AWS
+      specific settings, maintained with destinct profile names similar to GCE.
+      Similar to `$HOME/.config/gcloud`, contents should be protected and treated
+      as if they were passwords.
+
+## APIv1 (GCE Support)
+
+### Hack-script requirements
+
+1. Somewhere within the script the string `get_ci_vm APIv1` *must* be present.
+1. Any command-line arguments passed to the `hack/get_ci_vm.sh` script *must* be
+   passed as-is into the container as arguments, with the following exceptions:
+   1. When called with the `--setup` option, the script should:
+      1. Confirm `$GET_CI_VM` is non-zero.  This signifies the script is (now)
+         executing from **within** a newly created VM with CWD undefined.
+      1. Execute any commands necessary (on the VM) to prepare it for
+         use by the user.  For example, this could include calling *make* targets
+         and/or environment-setup scripts like `./contrib/cirrus/setup.sh`.
+      1. Exit non-zero if there is an error.
+   1. When called with the `--config` option, the script *must*:
+      1. Confirm `$GET_CI_VM` is `1`.  This signifies the script is (now)
+         executing from **within** the get_ci_vm container context.
+      1. Print on *stdout*, a list of `key=value` env. var lines conforming
+         to [the *APIV1 Env. Vars.* section below](README.md#env-vars).
+      1. Exit non-zero if there is an error.
+
+### Env. Vars.
+
+When `hack/get_ci_vm.sh` is executed with the `--config` argument, it
+*must* emit a `key=value` list of env. vars. to *stdout* if and only if
+the `$GET_CI_VM` value is set to `1`.
```

Comment:
Again, I don't see any code checking for `1`, just for nonzero.

Reply:
This check is done in the hack script itself, [example from podman.](https://github.com/containers/podman/blob/main/hack/get_ci_vm.sh#L31-L53)  I'll see if I can make this more clear in the docs.